src/ray/core_worker/store_provider/memory_store/memory_store.cc

```diff
@@ -82,8 +79,7 @@ GetRequest::GetRequest(absl::flat_hash_set<ObjectID> object_ids,
     : object_ids_(std::move(object_ids)),
       num_objects_(num_objects),
       remove_after_get_(remove_after_get),
-      abort_if_any_object_is_exception_(abort_if_any_object_is_exception_),
-      is_ready_(false) {
+      abort_if_any_object_is_exception_(abort_if_any_object_is_exception_) {
   RAY_CHECK(num_objects_ <= object_ids_.size());
```

Comment:
nit: `RAY_CHECK_LE`, which prints out the value if mismatch
not your responsibility, no need to change in this PR